katalog/calico/sm.yml

```diff
@@ -0,0 +1,32 @@
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  labels:
+    k8s-app: calico-node
+  name: calico-node
+  namespace: monitoring
+spec:
+  endpoints:
+  - interval: 15s
+    port: metrics
+  jobLabel: calico-node
```

Comment:
This should read `jobLabel: k8s-app` since it should point to the label key and not the label value.